### PR TITLE
Fix/cfe 122 fix unsupported token card

### PR DESCRIPTION
**Pull request by jennieramida**

### CODE REVIEW

File: src/lib/components/copy/CopyTemplate.tsx [hunk 1]
```diff
@@ -24,15 +24,16 @@ export const CopyTemplate = ({
 
   return (
     <Tooltip isOpen={!isDisabled && hasCopied} label={copyLabel}>
-      <Box
+      <Flex
+        direction="column"
```
Comment (songwongtp) [inline src/lib/components/copy/CopyTemplate.tsx:28]: isn't this the same as `Box`?

File: src/lib/components/modal/UnsupportedTokensModal.tsx [hunk 2]
```diff
@@ -58,35 +58,52 @@ const UnsupportedToken = ({ token }: { token: TokenWithValue }) => {
...
+          gap={1}
+          minH={6}
+          // order={{ base: 2, md: 1 }}
+        >
+          <Text
+            variant="body2"
+            className={isMobile ? "" : "ellipsis"}
+            wordBreak="break-all"
+          >
+            {isMobile ? token.denom : getTokenLabel(token.denom, token.symbol)}
```
Comment (songwongtp) [inline src/lib/components/modal/UnsupportedTokensModal.tsx:82]: why don't we care about `token.symbol` on mobile?

File: src/lib/components/modal/UnsupportedTokensModal.tsx [hunk 3]
```diff
@@ -58,35 +58,52 @@ const UnsupportedToken = ({ token }: { token: TokenWithValue }) => {
...
       >
-        <Flex gap={1} alignItems="center">
-          <Text variant="body2" className="ellipsis">
-            {getTokenLabel(token.denom, token.symbol)}
+        <Flex
+          alignItems="center"
+          justifyContent="center"
+          gap={1}
+          minH={6}
+          // order={{ base: 2, md: 1 }}
```
Comment (songwongtp) [inline src/lib/components/modal/UnsupportedTokensModal.tsx:75]: remove

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: src/lib/components/copy/CopyTemplate.tsx (hunk 1 above)
Comment (jennieramida) [reply src/lib/components/copy/CopyTemplate.tsx:28]: not exactly, somehow alignment does work only with flex

**Review by evilpeach — APPROVED** [review]

**Review by songwongtp — APPROVED** [review]

**Pull request merged**